platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/LinkedEventIntakeScheduler.java

```diff
@@ -17,25 +17,30 @@
 package com.swirlds.platform.wiring;
 
 import com.swirlds.common.wiring.InputWire;
+import com.swirlds.common.wiring.OutputWire;
 import com.swirlds.common.wiring.TaskScheduler;
 import com.swirlds.common.wiring.WiringModel;
 import com.swirlds.platform.components.LinkedEventIntake;
+import com.swirlds.platform.internal.ConsensusRound;
 import com.swirlds.platform.internal.EventImpl;
 import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.List;
 
 /**
  * Wiring for the {@link LinkedEventIntakeScheduler}.
  */
 public class LinkedEventIntakeScheduler {
-    private final InputWire<EventImpl, Void> eventInput;
+    private final InputWire<EventImpl, List<ConsensusRound>> eventInput;
+
+    private final OutputWire<ConsensusRound> eventOutput;
```

Comment:
I'm feeling like the second parameter of the input wire is useless noise if it doesn't have a tight correspondence with an output wire.   For example in 

```
public class EventDeduplicatorScheduler {
    private final TaskScheduler<GossipEvent> taskScheduler;
    private final InputWire<GossipEvent, GossipEvent> eventInput;
    private final InputWire<Long, GossipEvent> minimumGenerationNonAncientInput;
```

The `GossipEvent` parameter of the minGenNonAncient input wire is completely nonsense.   It makes it look like a long goes in and a single GossipEvent is supposed to come out.  

If the schedulers/components are not strongly typed with output wire types, then it is just noise on the input wire's type because it is not forced to correspond to anything, it only corresponds by convention and we're breaking the convention in many places.

Reply:
Much of my commentary here is on the framework itself now that I see it being applied to a real world context and not toy examples.

Reply:
Note that I intend to make the simplification we discussed with wires soon (i.e. where we hide the second generic argument of an input wire).